lib/api/queries/crowdaction_queries.dart

```diff
@@ -0,0 +1,5 @@
+const placeHolder = "#i#";
+
+const getCrowdActionsQuery = "{ crowdactions {name description start end } }";
```

Comment:
I'm of no actual opinion. As long as we are being descriptive enough in our naming, I used action instead of type, but we can definitely do `type` first for variables and `action` for methods. 

I do it out of habit like this because when you include an action inside a name for a method it can get tricky fast.
```
FetchCrowdActions();
CrowdActionsFetch();
-
GetCrowdActions();
CrowdActionsGet();
```